recce_cloud/cli.py

```diff
@@ -430,5 +432,137 @@ def download(target_path, session_id, prod, dry_run, force):
         download_with_platform_apis(console, token, ci_info, target_path, force)
 
 
+@cloud_cli.command()
+@click.option(
+    "--repo",
+    type=str,
+    help="Repository full name (owner/repo). Auto-detected from git remote if not provided.",
+)
+@click.option(
+    "--since",
+    type=str,
+    default="30d",
+    help="Start date (ISO format 2024-11-01 or relative like 30d). Default: 30d",
+)
+@click.option(
+    "--until",
+    type=str,
+    default=None,
+    help="End date (ISO format or relative). Default: today",
+)
+@click.option(
+    "--base-branch",
+    type=str,
+    default="main",
+    help="Target branch filter. Default: main",
+)
+@click.option(
+    "--merged-only/--include-open",
+    default=True,
+    help="Only include merged CRs (default) or include open CRs too",
+)
+@click.option(
+    "-o",
+    "--output",
+    type=click.Path(),
+    default=None,
+    help="Output file path for CSV. Default: no file output",
+)
+@click.option(
```

Comment:
I feel the purpose of `--csv` is to output CSV format at first.

```
  -o, --output PATH               Output file path for CSV. Default: no file
                                  output
  --csv                           Output CSV to stdout instead of summary
```